tests/compliance/blackduck-spec.js

```diff
@@ -0,0 +1,176 @@
+'use strict'
+
+const rewire = require('rewire')
+const sinon = require('sinon')
+const expect = require('chai').expect
+const Promise = require('promise')
+const logger = require('../../lib/logger')
+const blackduck = rewire('../../lib/compliance/blackduck')
```

Comment:
Since we're using `rewire`, we shouldn't need to have those dependencies included in the module in the `lib/compliance/blackduck` module, since they can be stubbed even if they at the module scope.